conda/plugins/manager.py

```diff
@@ -366,6 +373,14 @@ def get_subcommands(self) -> dict[str, CondaSubcommand]:
             for subcommand in self.get_hook_results("subcommands")
         }
 
+    def run_post_transaction_hooks(self, action: _Action) -> None:
+        """Run the post transaction hooks for the given action.
+
+        :param action: Action for which post transaction hooks are to be run.
+        """
+        for hook in self.get_hook_results("post_transactions"):
+            hook.run(action)
```

Comment:
```suggestion
            if isinstance(action, hook.action_type):
                hook.run(action)
```

I don't think this would help much, though... We are just skipping one call.

Reply:
It'll help if the hook is configured to be run only on very specific actions, at least :shrug: